files/dashboards/openshift-logging-dashboard.json

```diff
@@ -1755,5 +2061,5 @@
   "timezone": "",
   "title": "Openshift Logging",
   "uid": "seyA3sMMk",
-  "version": 1
-}
+  "version": 2
+}
```

Comment:
@ajaygupta978 more lines to get back to original here in the end :-)

Reply:
Done.